tests/cli/workflow/nf_analysis/test_cli_report_deliver.py

```diff
@@ -1,33 +1,33 @@
 """Tests CLI common methods to create the report_deliver for NF analyses."""
 
 import logging
-import pytest
 from pathlib import Path
 
-from click.testing import CliRunner
+import pytest
 from _pytest.logging import LogCaptureFixture
+from click.testing import CliRunner
 
+from cg.cli.workflow.base import workflow as workflow_cli
+from cg.constants import EXIT_SUCCESS, Workflow
+from cg.constants.constants import FileFormat
+from cg.io.controller import ReadFile
 from cg.meta.workflow.nf_analysis import NfAnalysisAPI
-from cg.cli.workflow.rnafusion.base import report_deliver
 from cg.models.cg_config import CGConfig
-from cg.constants import EXIT_SUCCESS
-from cg.io.controller import ReadFile
-from cg.constants.constants import FileFormat
 
 LOG = logging.getLogger(__name__)
 
 
 @pytest.mark.parametrize(
-    "context",
-    ["rnafusion_context", "taxprofiler_context"],
+    "workflow",
+    [Workflow.RNAFUSION, Workflow.TAXPROFILER],
 )
-def test_report_deliver_without_options(cli_runner: CliRunner, context: CGConfig, request):
+def test_report_deliver_without_options(cli_runner: CliRunner, workflow: Workflow, request):
```

Comment:
```suggestion
def test_report_deliver_without_options(cli_runner: CliRunner, workflow: Workflow, request: FixtureRequest):
```
I think this is the typehint for it